base/src/main/java/org/aya/core/term/ElimTerm.java

```diff
@@ -27,6 +32,13 @@ public sealed interface ElimTerm extends Term {
       if (hole.args().sizeLessThan(hole.ref().telescope))
         return new CallTerm.Hole(hole.ref(), hole.ulift(), hole.contextArgs(), hole.args().appended(app.arg()));
     }
+    if (app.of() instanceof ErasedTerm erased) {
+      if (erased.type() instanceof FormTerm.Pi pi) {
```

Comment:
Probably replace with assertion... This should always hold

Reply:
Maybe erased.type() could be an ErrorTerm

Reply:
Oh

Reply:
Can you add this to the comment?

Reply:
ok